internal/provider/resource_auth_method_oidc.go

```diff
@@ -267,8 +267,12 @@ func resourceAuthMethodOidcCreate(ctx context.Context, d *schema.ResourceData, m
 	if clientSecret, ok := d.GetOk(authmethodOidcClientSecretKey); ok {
 		opts = append(opts, authmethods.WithOidcAuthMethodClientSecret(clientSecret.(string)))
 	}
-	if maxAge, ok := d.GetOk(authmethodOidcMaxAgeKey); ok {
+	// null values are not correctly recognized by the Terraform SDK, so we instead check here for maxAge value
```

Comment:
This wording is a bit confusing to me, by "null values", do we mean an integer with a value of zero? Do we mean an omitted value?

Reply:
Null values in this context means if the value isn't provided in the state altogether - if you go from having a maxAge of 0 to removing it completely from a Terraform `.tf` config file, the state still treats maxAge as existing as 0

Reply:
OK, null or "omitted" sounds correct here then 👍🏻. I've seen "null value" used to mean "the zero value of the type", which is why I asked.